vmm/src/memory_manager.rs

```diff
@@ -232,6 +244,27 @@ impl MemoryManager {
             start_of_device_area = start_of_device_area.unchecked_add(size);
         }
 
+        let mut virtiomem_region = None;
+        let mut virtiomem_resize = None;
+        if let Some(size) = virtiomem_size {
+            let block_size = vm_virtio::VIRTIO_MEM_DEFAULT_BLOCK_SIZE as u64;
+            let start_addr = if start_of_device_area < arch::layout::MEM_32BIT_RESERVED_START {
+                arch::layout::RAM_64BIT_START
+            } else {
+                GuestAddress((start_of_device_area.0 + block_size - 1) / block_size * block_size)
+            };
+            virtiomem_region = Some(MemoryManager::create_ram_region(
+                backing_file,
+                start_addr,
+                size as usize,
+            )?);
+
+            virtiomem_resize = Some(vm_virtio::Resize::new().map_err(Error::EventFdFail)?);
+
+            start_of_device_area = start_addr.unchecked_add(size);
+        }
+        let virtiomem_region_c = virtiomem_region.clone();
```

Comment:
This isn't the style. If you need to clone to it when you instantiate the struct.

Reply:
Fixed.